doc/howto_release.md

```diff
@@ -301,42 +302,29 @@ Software pages:
 - wiki updates, only when new major release:
     - {{cmd|xxxx}} macro: <https://grasswiki.osgeo.org/wiki/Template:Cmd>
     - update last version on main page
-- trac updates, only when new major release:
-    - Set "complete" flag in <https://trac.osgeo.org/grass/milestone/7.8.x> -->
-      Edit Milestone
-        - also: Retarget associated open tickets to milestone 7.8.x
-    - Batch modify tickets, set to next milestone (update this query
-      accordingly: two entries to change):
-        - <https://trac.osgeo.org/grass/query?status=assigned&status=new&status=reopened&milestone=7.8.0&milestone=7.8.1&group=status&col=id&col=summary&col=owner&col=type&col=priority&col=component&col=version&order=priority>
-    - Set max items to 1000, then select all shown tickets via Status:
-      assigned/new/reopened sections
-    - Scroll down to "Batch modify": under the "Comment" section, add Field
-      "Milestone" and set to next version
-    - then use "Change ticket" button, done.
-        - Only in case of new release branch being created:
-- Add Wiki Macro definitions for manual pages G7X:modulename
+- Add trac Wiki Macro definitions for manual pages G8X:modulename
     - Edit: <https://trac.osgeo.org/grass/wiki/InterMapTxt>
 
 #### WinGRASS notes
 
 - Update grass_packager_release.bat, eg.
 
 ```
-     set MAJOR=7
-     set MINOR=8
+     set MAJOR=8
+     set MINOR=0
      set PATCH=0RC1
 ```
 
 - Update addons (grass_addons.sh) rules, eg.
 
 ```
-     compile $GIT_PATH/grass7 $GISBASE_PATH/grass780RC1   $ADDON_PATH/grass780RC1/addons
+     compile $GIT_PATH/grass8 $GISBASE_PATH/grass800RC1  $ADDON_PATH/grass800RC1/addons
```

Comment:
good to have this in place but the `grass8` branch in the addons repo has not been created yet

Reply:
True but @wenzeslaus planned to do that these days (see https://lists.osgeo.org/pipermail/grass-dev/2021-November/095449.html). I'd prefer to get rid of G7.x in this document.

Reply:
Created, so yes, it's the time for updates.